operations/research-and-development/product/product-management-team-handbook.md

```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
-  * Get Jira training from PM buddy 
+  * Get Jira training from PM buddy
     * [Customer Requests](https://mattermost.atlassian.net/projects/CR/issues/CR-16?filter=allopenissues)
     * [Mattermost Platform](https://mattermost.atlassian.net/projects/MM/issues/MM-10977?filter=allopenissues)
-* Make your first edit and pull request on Github
+* Make your first edit and pull request on GitHub
   * Update documentation or product help text
 * Small contributions
   * File a bug in Jira
   * Test a new feature on a test server, and suggest an improvement
-  * Leave feedback on a design proposal 
+  * Leave feedback on a design proposal
 
 Readings:
 
-* The Advantage: 
+* The Advantage:
   * [https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view](https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view)
 * The Five Dysfunctions of a Team: A Leadership Fable
-  * [https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing](https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing) 
+  * [https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing](https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing)
 
 ### First 30 Days
 
 Objective: Build foundational knowledge
 
-Training Focus:
+Training focus:
 
-* Users & customers 
-* Product 
+* Users and customers
+* Product
   * Features/Functions
   * Technical
-* Market & Competitive Landscape
+* Market and Competitive Landscape
 * Team
   * People and Processes
 
 Activities:
 
-* Learn about our customers and users 
-  * Support 
-    * Customers: Zendesk \(Paul\) 
-      * Meet with Paul for Zendesk training and support overview 
-      * Join support standups for a week or two, and help out on support tickets 
+* Learn about our customers and users
+  * Support
+    * Customers: Zendesk \(Sven\)
+      * Meet with Sven for Zendesk training and support overview
+      * Join support standups for a week or two, and help out on support tickets
     * Community: GitHub, Forums \(Amy\)
-  * Customer success \(Meet with a CSM - Emily Hollinger, Stu Doherty, Rosie Lane\) 
+  * Customer success \(Meet with a CSM - Emily Hollinger, Stu Doherty, Rosie Lane\)
     * Overview of Customer Success team, who owns what
     * Current Customer Success processes
     * Typical challenges during customer onboarding
     * Join one CSM team meeting to meet the rest of the CS team
   * Customer calls
-    * Join calls with CSMs 
-    * Join calls with Sales team 
+    * Join calls with CSMs
+    * Join calls with Sales team
   * Feature requests
-    * Uservoice forums - [https://mattermost.uservoice.com/](https://mattermost.uservoice.com/)  
+    * Uservoice forums - [https://mattermost.uservoice.com/](https://mattermost.uservoice.com/)
     * [Customer request triage](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning.nodetail&quickFilter=97) 
```

Comment:
We should replace this link with "Process feedback in productboard" with a link to productboard inbox: https://mattermost.productboard.com/insights/shared-inbox